### DISCUSSION TITLE
Grbl and Ortur keeps stopping midway laser engraving project

### DISCUSSION THREAD

**Discussion by DAwoodworking** [arkypita/LaserGRBL#1420] [Q&A]
See pics attached.
the LaserGbrl and Ortur keeps stopping mid print and have to restart the program and wasted several nice wood projects that i was trying to engrave.
We are new to this, so it showed to send the attached pictures to this group to try to unersatnd the issue and what i can do to fix this.
Thank you in advance for your help

![20210325_003132860_iOS](https://user-images.githubusercontent.com/81364237/112470224-cc4db480-8d40-11eb-961e-af7f1cba8677.jpg)
![20210325_012851313_iOS](https://user-images.githubusercontent.com/81364237/112470234-cf48a500-8d40-11eb-99b2-3fe0c56aa5b6.jpg)
![20210325_012900804_iOS](https://user-images.githubusercontent.com/81364237/112470242-d2dc2c00-8d40-11eb-95bd-08fcde642a22.jpg)

**Comment by arkypita** [comment]
Hi
LaserGRBL is open software not related to any specific laser engraver hardware or vendors.
I suggest you to contact the manufacturer of your engraver for specific and specialized help.

- [Ortur user group](https://www.facebook.com/groups/orturusergroup) (community)
- [Ortur customer support](https://ortur.tech/support/)

In meanwhile  make sure you have read the [LaserGRBL FAQ](https://lasergrbl.com/faq) where you can find the answers to the most common questions.

**Comment by arkypita** [comment]
PS: i guess your PC USB port/connector has issue.

**Comment by Milan-As** [comment]
I solved the same problem. Replacement of test cables from XP, W7, W10 different crossings. This resulted in suspicions of HW and SW cutters. Finally, a test of various CH340 drivers helped. For W10 installation CH341ALL.

**Comment by arkypita** [comment]
Ortur does not have CH340 chip

**Comment by leelouric** [comment]
i fixed the problem in windows to disable usb power saving after 14 hours engraving he still run

**Comment by arkypita** [comment]
Hi @leelouric 
I will updata lasergrbl faq with your hint.

Can you put some screenshoot and description of how to configure usb power saving?
